finary_uapi/__main__.py

```diff
@@ -377,7 +379,10 @@ def main() -> int:  # pragma: nocover
         elif args["loans"]:
             result = get_loans(session)
         elif args["credit_accounts"]:
-            result = get_credit_accounts(session)
+            if args["transactions"]:
+                result = get_portfolio_creditcards_transactions(session)
```

Comment:
Could you rename `get_portfolio_creditcards_transaction` to `get_portfolio_credit_accounts_transaction` to keep the naming the same for all functions ? Thanks !